src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.cs

```diff
@@ -99,5 +102,50 @@ public LambdaComparer(Func<T?, T?, bool> equal, Func<T?, int> getHashCode)
 
             public int GetHashCode(T obj) => _getHashCode(obj);
         }
+
+        private const string RegexGeneratorAttributeSource =
+@"#nullable enable
+using global::System.Diagnostics;
+using global::System.Diagnostics.CodeAnalysis;
+using global::System.Threading;
+ 
+namespace System.Text.RegularExpressions
+{ 
+    /// <summary>Instructs the System.Text.RegularExpressions source generator to generate an implementation of the specified regular expression.</summary>
+    /// <remarks>The generator associated with this attribute only supports C#.  It only supplies an implementation when applied to static, partial, parameterless, non-generic methods that are typed to return <see cref=""Regex""/>.</remarks>
+    [Conditional(""INCLUDE_SYSTEM_TEXT_REGULAREXPRESSIONS_REGEXGENERATORATTRIBUTE"")]
```

Comment:
If someone wanted to keep the attributes in the assembly, for say auditing purposes, the intention is they would set this define in their .csproj, right?